front/components/app/blocks/Output.tsx

```diff
@@ -239,27 +239,34 @@ export function Execution({
     <div className="flex flex-auto flex-col overflow-hidden">
       {trace.map((t, i) => {
         return (
-          <div key={i} className="flex-auto flex-col">
-            {t.error != null ? (
-              <div className="flex flex-auto flex-row">
-                <ExclamationCircleIcon className="mt-0.5 flex h-4 w-4 text-red-400" />
-                <Error error={t.error} />
-              </div>
-            ) : (
-              <div className="flex flex-row">
-                <div className="flex flex-initial">
-                  <CheckCircleIcon className="min-w-4 mt-0.5 h-4 w-4 text-emerald-300" />
+          <div>
+            <div key={i} className="flex-auto flex-col">
+              {t.error != null ? (
+                <div className="flex flex-auto flex-row">
+                  <ExclamationCircleIcon className="mt-0.5 flex h-4 w-4 text-red-400" />
+                  <Error error={t.error} />
                 </div>
-                <div className="flex flex-1">
-                  <ValueViewer
-                    block={block}
-                    value={t.value}
-                    topLevel={true}
-                    k={null}
-                  />
+              ) : (
+                <div className="flex flex-row">
+                  <div className="flex flex-initial">
+                    <CheckCircleIcon className="min-w-4 mt-0.5 h-4 w-4 text-emerald-300" />
+                  </div>
+                  <div className="flex flex-1">
+                    <ValueViewer
+                      block={block}
+                      value={t.value}
+                      topLevel={true}
+                      k={null}
+                    />
+                  </div>
                 </div>
-              </div>
-            )}
+              )}
+            </div>
+            <div>
```

Comment:
Cool, we'll need to iterate on that as well.

I think we want to separate the outputs from the console log (they can be expanded separately) so that the logs don't interfere with the base UX of the output of each block. This is already a great start (means the console.log will be outside the boundary of the block but this is fine).

Then it's a question of styling things a bit so that it looks nice and probably have the logs shown in a textarea with monospaced fonts

Reply:
Only visible for the code block of course